app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
 <h1>Here you will find a list of all the users</h1>
 
+<div class='users-container'>
+  <% @users.each do |user| %>
```

Comment:
**OPTIONAL**
- Kindly consider using partials in order to make your code less and not to use `@objects.each` more than once! 📔 

```
    # app/views/users/_user.html.erb


     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```